src/OpenLoco/Windows/StationList.cpp

```diff
@@ -56,11 +56,11 @@ namespace OpenLoco::Ui::Windows::StationList
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_road_stations),
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_airports),
         makeRemapWidget({ 3, 15 }, { 31, 27 }, WidgetType::tab, WindowColour::secondary, ImageIds::tab, StringIds::tooltip_ship_ports),
-        makeWidget({ 0, 14 }, { 26, 26 }, WidgetType::buttonWithImage, WindowColour::primary, StringIds::null, StringIds::tooltip_select_company),
```

Comment:
Looks like the station list and vehicle windows were the only ones affected by the issue? Probably worth a changelog entry, mentioning them by name.

Reply:
The other one was the title screen

Reply:
> The other one was the title screen

Then this would fix #293

Reply:
Yes that's definitely related